manager/controllers/systems.go

```diff
@@ -37,6 +37,7 @@ type SystemItemAttributes struct {
 	RhbaCount      int        `json:"rhba_count" csv:"rhba_count" query:"sp.advisory_bug_count_cache"`
 	RheaCount      int        `json:"rhea_count" csv:"rhea_count" query:"sp.advisory_enh_count_cache"`
 	Stale          bool       `json:"stale" csv:"stale" query:"sp.stale"`
+	ThirdParty     bool       `json:"third_party" csv:"third_party" query:"sp.third_party"`
```

Comment:
Add it also to advisory_systems.go and its test please.

Reply:
updated TestAdvisorySystemsDefault